docs/dev/libvirt/README.md

```diff
@@ -30,7 +30,7 @@ repository to ensure you get a new enough version of qemu-kvm.
 On Fedora, CentOS/RHEL:
 
 ```sh
-sudo yum install libvirt libvirt-devel libvirt-daemon-kvm qemu-kvm
+sudo yum install libvirt-devel libvirt-daemon-kvm
```

Comment:
I would add `libvirt-client` to that list, or add a mention that it's needed if you need `virsh`. Or is it already a dependency of `libvirt-devel`/`libvirt-daemon-kvm`?

Reply:
Yes, good point, I've done that